src/client/utils/dataset/dataset.ts

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2015-2016 Imply Data, Inc.
+ * Copyright 2017-2018 Allegro.pl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Timezone } from "chronoshift";
+import { Dataset, Datum, TimeRange } from "plywood";
+import { formatValue } from "../../../common/utils/formatter/formatter";
+import { SPLIT } from "../../config/constants";
+
+export type Order<D> = (a: [string, number, D], b: [string, number, D]) => number;
+
+export const orderByValueDecreasing: Order<any> = ([_, countA], [__, countB]) => {
+  if (countA < countB) {
+    return 1;
+  }
+
+  if (countA > countB) {
+    return -1;
+  }
+
+  return 0;
+};
+
+export const orderByValueIncreasing: Order<any> = (a, b) => {
+  return -orderByValueDecreasing(a, b);
+};
+
+export const orderByTimeDimensionDecreasing: Order<TimeRange> = ([_, __, originalA], [___, ____, originalB]) => originalA.compare(originalB);
+export const orderByTimeDimensionIncreasing: Order<TimeRange> = ([_, __, originalA], [___, ____, originalB]) => -originalA.compare(originalB);
+
+export const fillDataset = (dataset: Dataset, measureName: string, secondSplitName: string, order: Order<any>, timezone: Timezone): Dataset => {
```

Comment:
function and file names seems very generic

btw. do we need to return plywood dataset? POJsO wouldn't suffice?

Reply:
Using Dataset turned out a bit easier for testing - Dataset.toJS puts additional properties on objects - I would have to copy them all manually to my fixtures.

Reply:
Changed the name btw